src/main.js

```diff
@@ -29,10 +35,19 @@ const renderTasks = () => {
   dataTasksWithoutGropOnLoad.forEach((task) => addToDoSection.append(ToDoItem(task)));
 };
 
+const setTheme = () =>
+  document.body.classList.add(`${isDarkThemeModeOnLoad ? 'theme-dark' : 'theme-light'}`);
+
+const accordions = document.querySelector('.accordion-navbar-container');
+console.log('accordions before initialisation', accordions);
 const initializeFrontendApp = () => {
+  setTheme();
```

Comment:
You actually do not have to ivoke this method inside initialize function as body is accessible outside, but for the sake of cinsistancy let leave it as it is

Reply:
One las UI thing - dark tekst witl **low contrast** inside INPUT on DARK MODE 



![obraz](https://user-images.githubusercontent.com/21220135/235112646-3311f1c5-8c6d-4bf5-b9d2-4652e0155f8e.png)